### PR TITLE
fix, restart port-fw on node restart

### PR DESCRIPTION
**Pull request by pepoviola**
Recreate the port-fw when a node is restarted.

### CODE REVIEW

File: src/networkNode.ts [hunk 1]
```diff
@@ -77,7 +83,26 @@ export class NetworkNode implements NetworkNodeInterface {
...
+          "{{PORT}}",
+          fwdPort.toString(),
+        );
+
+        this.apiInstance = undefined;
+      }
+
+      return true;
+    }
+    return false;
```
Comment (wirednkod) [inline src/networkNode.ts:105]: Its better to do an early exit
```suggestion
    if(result.exitCode !== 0) return false;
    // restart the port-fw if needed
    const url = new URL(this.wsUri);
    if (parseInt(url.port, 10) !== RPC_WS_PORT) {
      const fwdPort = await client.startPortForwarding(
        RPC_WS_PORT,
        this.name,
      );

    this.wsUri = WS_URI_PATTERN.replace("{{IP}}", LOCALHOST).replace(
      "{{PORT}}",
      fwdPort.toString(),
    );

    this.apiInstance = undefined;
    return true;
```

File: src/networkNode.ts [hunk 2]
```diff
@@ -179,6 +204,10 @@ export class NetworkNode implements NetworkNodeInterface {
         return;
       };
 
+      // // force connect
+      // if (this.apiInstance) (this.apiInstance as ApiPromise).disconnect();
+      // await this.connectApi();
```
Comment (wirednkod) [inline src/networkNode.ts:209]: Please remove commented code (If its a TODO thing, just add that to a comment )

File: src/networkNode.ts (hunk 1 above)
Comment (pepoviola) [reply src/networkNode.ts:105]: Yes, sounds better 👍

File: src/networkNode.ts (hunk 2 above)
Comment (pepoviola) [reply src/networkNode.ts:205]: Ahh nice catch, I will remove those lines. Thanks!

**Review by wirednkod — APPROVED** [review]

**Pull request merged**